src/localization/currency.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import currency from 'currency.js';
+import { LANGUAGE_CODES } from '.';
+
+const CURRENCY_CONFIGS = {
+  DEFAULT: {
+    decimal: '.',
+    separator: ',',
+    pattern: '!#',
+    symbol: '$',
+    formatWithSymbol: true,
+  },
+  [LANGUAGE_CODES.FRENCH]: {
```

Comment:
Nice! 

Definitely think its a good idea to localise decimals etc., only slight reservation is from a UI point of view re. the currency symbol and whether it might suggest to the user that we handle multiple currencies (as far as I'm aware we don't currently store transacts with any currency related metadata and don't handle conversions etc.).

Reply:
hmm I guess that's a good point - I wonder if it's better to just not use any symbol... hmm..